sled-agent/src/params.rs

```diff
@@ -216,7 +216,7 @@ pub struct Zpool {
 
 // The type of networking 'ASIC' the Dendrite service is expected to manage
 #[derive(
-    Clone, Debug, Deserialize, Serialize, JsonSchema, PartialEq, Eq, Copy, Hash,
+    Clone, Debug, Deserialize, Serialize, JsonSchema, PartialEq, Eq, Hash,
```

Comment:
It seems like this is still `Copy`, but I might be missing something.

Reply:
Can you elaborate?

Reply:
There's a diff here, removing the `Copy` trait. This enum sure looks like it's `Copy` to me, so I'm wondering if we want to put it back. It looks like maybe it was historical -- we added data to one of the enum variants that wasn't `Copy`; had to remove `Copy` from this enum; and then removed the data, but didn't put `Copy` back.

This is all tiny, and I'm just curious.